src/BenchmarkDotNet/Phd/BdnJob.cs

```diff
@@ -0,0 +1,8 @@
+using Perfolizer.Phd.Dto;
+
+namespace BenchmarkDotNet.Phd;
+
+public class BdnJob : PhdJob
```

Comment:
what is the point of having this empty and public class? Also, it's another thing called Job..

Reply:
> what is the point of having this empty and public class?

This is a work-in-progress solution, which is not fully ready yet. I've merged it to master mostly because of the Cpu/Os refactoring (it was a long-lasting branch and I was tired of solving merge conflicts). I guess it's a good idea to move it away from master to a feature branch until it's fully ready. **Will do it.**

> Also, it's another thing called Job.

The current design of jobs and configs has several problems. Here are two main problems I want to solve:
1. The current design with `CharacteristicObject` is overcomplicated, it's challenging to work with it. I want to come up with a simpler design. However, if I start refactoring jobs right now, we will get a lot of breaking changes. Instead, I want to build a new solution side-by-side with the existing one preserving the existing solution. When everything is ready, we can switch to the new API.
2. 
3. Current jobs are not deserializable. This means that once we export the benchmark results to files (with job descriptions inside), we can't load them back. But we need this information if we want to work with the results later. Manual recreation of the original jobs (and other config entities) based on the current xml/json exported files is quite challenging and, in some places, just impossible. In the new design, I want to have a set of extremely simple DTO entities, which can be easily serialized/deserialized at any time. My current attempt focuses on the `Phd*` classes (which provide some common ground with other tools) and the `Bdn*` classes (which extend `Phd*` with BenchmarkDotNet-specific properties).

Reply:
> it was a long-lasting branch and I was tired of solving merge conflicts

I understand. 

> The current design with CharacteristicObject is overcomplicated, it's challenging to work with it.

I agree. In case you have some idea you can express it as an issue with some pseudocode, I would be happy to provide feedback (I am slowly getting better at API design, this year I even got some APIs approved by the .NET API review with the first attempt).

Reply:
BTW I am not sure if we should be using the `Phd` word, as it will most likely be mapped to `PhD` by most of our users ;p